hugo/themes/genome-portal-kit/layouts/species_bundle/intro.html

```diff
@@ -0,0 +1,142 @@
+				{{ .Title }}
+			    </p>
+			{{ end }}
+		    {{ end }}
+		</figure>
+            </div>
+
+            <div class="col-lg-6 col-md-12">
+		{{ partial "distrib_map.html" . }}
+            </div>
+	</div>
+
+
+	<div class="row mt-3">
+
+            <!-- Lineage Info -->
+            <div class="col-lg-4 col-md-12 scilife-subsection">
+		<h3>Taxonomy</h3>
+		{{ with .Params.lineage }}
+		    {{ $context := dict
+		    "path" $.Path
+		    "lineage" .
+		    "taxonomy_ranks" (or $.Page.Params.taxonomy_ranks $.Site.Params.taxonomy_ranks)
+		    }}
+		    {{ partial "species_bundle/lineage.html" $context }}
+		{{ else }}
+		    {{ warnidf "missing-taxonomy" "Missing taxonmomy lineage for %s" .Path }}
+		{{ end }}
+
+            </div>
+
+            <div class="col scilife-subsection">
+		{{ if or .Params.iucn_category .Params.swe_red_list }}
+                    <h3>Vulnerability</h3>
+                    <p>
+			{{ if .Params.iucn_category }}
+                            <b>IUCN Category:</b>
+                            <a href="{{ .Params.iucn_link }}" target="_blank">{{ .Params.iucn_category }}</a>
+                            <br>
+			{{ end }}
+
+			{{ if .Params.swe_red_list }}
+                            <b>Swedish Red List:</b>
+                            <a href="{{ .Params.swe_red_list_link }}" target="_blank">{{ .Params.swe_red_list }}</a>
+                            <br>
+			{{ end }}
+                    </p>
+		{{ end }}
+
+
+		<h3>External links</h3>
+
+		<div class="d-grid gap-2 d-md-flex text-center text-md-left">
+                    {{ if .Params.gbif_taxon_id }}
+			<a href="https://species.biodiversitydata.se/species/{{ .Params.gbif_taxon_id }}" target="_blank">
+                            <button class="scilife-external-btn btn btn-lg">
+				<img src="/img/icons/location.svg" alt="Location Icon" class="scilife-icon">
+				<br>
+				Occurrence data in SBDI
+                            </button>
+			</a>
+
+			<a href="https://www.gbif.org/species/{{ .Params.gbif_taxon_id }}" target="_blank">
+                            <button class="scilife-external-btn btn btn-lg">
+				<img src="/img/icons/location.svg" alt="Location Icon" class="scilife-icon">
+				<br>
+				Occurrence data in GBIF
+                            </button>
+			</a>
+                    {{ end }}
+
+                    {{ if .Params.goat_webpage }}
+			<a href="{{ .Params.goat_webpage }}" target="_blank">
+                            <button class="scilife-external-btn btn btn-lg">
+				<img src="/img/icons/db.svg" alt="Database Icon" class="scilife-icon">
+				<br>
+				Record in Genomes on a Tree (GoaT)
+                            </button>
+			</a>
+                    {{ end }}
+
+		</div>
+
+	    </div>
+
+
+
+	    <div class="row">
+
+	    </div>
+
+	    <div class="row">
+		<div class="col scilife-subsection mt-3">
+		    {{ .Content }}
+		</div>
+	    </div>
+
+	    {{ partial "last_updated.html" . }}
```

Comment:
That is a tricky question, part of a more general one: isn't it unwieldy to have two versions of every layout as is currently the case (and would be compounded by the addition of `layouts/partials/species_bundle/last_updated.html` This is where this refactoring shows its weakness and rudimentary state.

A possible solution to this would be not to define whole page layouts in the `genome-portal-kit` module, but only expose smaller partials that the main user site would include.

In this case, that would roughly mean removing `{{ partial "last_updated.html" }}`, and leaving it to the user to render the last modified date, providing the standard Hugo `lastmod`.

See issue [EB-376](https://scilifelab.atlassian.net/browse/EB-376) for a very rough sketch of this design.

Reply:
Oh, and I would leave it broken for now (until I can allot more work to this topic), as a reminder if need be of the brittle state of things.